packages/execution-environments/src/index.test.ts

```diff
@@ -0,0 +1,8 @@
+import * as mainExports from '.';
+
+describe('index.ts', () => {
+  it('has expected exports', () => {
+    expect(Object.keys(mainExports)).toHaveLength(1);
```

Comment:
It'll break as soon as we add something. Is there a need for this specific `expect`?

Reply:
I added these tests for coverage purposes, and I needed to `expect` something in the test case. We can modify it later to something more permissive.